vendor/plugins/edu.harvard.dce.paella.captionsPlugin/captionsPlugin.js

```diff
@@ -398,6 +402,22 @@ Class ("paella.plugins.DceCaptionsPlugin", paella.ButtonPlugin,{
 	        		paella.player.videoContainer.seekToTime(parseInt(secBegin));
 	        });
     	});
+    },
+
+    _addTagHeader: function(container, tags) {
+      var self = this;
+      if (!tags) return;
+      if ( ((Array.isArray && Array.isArray(tags)) || (tags instanceof Array)) == false) {
```

Comment:
my reading of this line is that it evaluates to true if `Array.isArray(tags)` and also if `tags` is not an instance of `Array`, i.e. `tags = [tags]` is always going to be executed.